aws/resource_aws_emr_cluster_test.go

```diff
@@ -98,6 +98,17 @@ func TestAccAWSEMRCluster_basic(t *testing.T) {
 					resource.TestCheckResourceAttr(resourceName, "bootstrap_action.#", "0"),
 				),
 			},
+			{
+				Config: testAccAWSEmrClusterConfigMultipleSubnets(rName),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckAWSEmrClusterExists(resourceName, &cluster),
+					resource.TestCheckResourceAttr(resourceName, "scale_down_behavior", "TERMINATE_AT_TASK_COMPLETION"),
+					resource.TestCheckResourceAttr(resourceName, "step.#", "0"),
+					resource.TestCheckResourceAttrSet(resourceName, "arn"),
+					resource.TestCheckNoResourceAttr(resourceName, "additional_info"),
+					resource.TestCheckResourceAttr(resourceName, "bootstrap_action.#", "0"),
```

Comment:
This should test that `ec2_attributes.0. subnet_ids` has the correct number of values and that the correct values are set and that `ec2_attributes.0. subnet_id` is not set. `TypeSet` values can be checked using `resource.TestCheckTypeSetElemAttrPair`.

In addition, in the first test step, the test should confirm that `ec2_attributes.0. subnet_id` is set to the correct value and that `ec2_attributes.0. subnet_ids` has zero elements.